template.ecosystem.json

```diff
@@ -1,3 +1,6 @@
+// This is a template for your ecosystem.json file. Refer to the wiki to learn how to set it up. 
+// (Note: you must delete "template." for it to work)
+
```

Comment:
I appreciate the effort, but you can't add comments on a `.json` file. People that don't know will have a problem launching the bot if they didn't delete this before it.

Reply:
You're right about that, that was my mistake.